### ISSUE TITLE
`fisher -V` doesn't allow dot characters

### ISSUE THREAD

**Issue by xnuk** [fisherman/fisherman#2]
![screenshot at 01-02 054709](https://cloud.githubusercontent.com/assets/3071003/12072155/c33322a2-b115-11e5-869f-385384664988.png)

**Comment by bucaran** [comment]
Good find. It does support `.` dots however. The problem is the oddly named theme `cmorrel.com` fails because of the regex: 

```
/[A-Za-z0-9_-]/
```

I think we could safely change this to:

```
/[A-Za-z0-9]([._-]?[a-z0-9]+)/
```

I will add it to the next milestone.

**Comment by bucaran** [comment]
EDIT: This will be fixed in 0.2.0, in the meantime, if you want to use `cmorrell.com` theme, use the full URL:

```
fisher install https://github.com/oh-my-fish/theme-cmorrell.com
```
